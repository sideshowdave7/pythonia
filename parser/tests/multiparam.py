def foo(a, b, c, d):
  print("hi")

foo(1, 2, 3, 4)
