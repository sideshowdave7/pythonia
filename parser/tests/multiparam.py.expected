(program
 (def (foo a b c d) (suite (expr (indexed print (called "hi")))))
 (expr (indexed foo (called 1 2 3 4))))

